daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v2.md

```diff
@@ -0,0 +1,165 @@
+---
+type: docs
+title: "PostgreSQL"
+linkTitle: "PostgreSQL"
+description: Detailed information on the PostgreSQL state store component
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-postgresql-v2/"
+  - "/operations/components/setup-state-store/supported-state-stores/setup-postgres-v2/"
+---
+
+{{% alert title="Note" color="primary" %}}
+This is the v2 of the PostgreSQL state store component, which contains some improvements to performance and reliability.
```

Comment:
CockroachDB will work but not sure if it should have its own component?

As for YugabyteDB, our tests show that it requires a bit of work at this stage. Specifically we need to make users enable the crypto extension for using the UUID generator function

Reply:
@ItalyPaleAle. A suggestion and comment

- "This is the v2 of the PostgreSQL state store component, which contains some improvements to performance and reliability. It is recommended to use the V2 component over the V1."

- Regarding "CockroachDB will work but not sure if it should have its own component?" - Should users ever use this component for CockroachDB then, or stay with the specific component https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-cockroachdb/ ? Not sure what you mean by "CockroachDB will work but not sure if it should have its own component?" since it already has its own component.